tests/cypress/e2e/sub-project-dependencies.cy.js

```diff
@@ -0,0 +1,34 @@
+describe('SubProjectDependencies', () => {
+  it('loads the dependency graph', () => {
+    cy.visit('viewSubProjectDependenciesGraph?project=SubProjectExample&date=2009-08-06%2012:19:56');
+
+    cy.get('#chart_placeholder').should('have.descendants', 'svg');
+  });
+
+
+  it('can interact with the dependency graph', () => {
+    cy.visit('viewSubProjectDependenciesGraph?project=SubProjectExample&date=2009-08-06%2012:19:56');
+
+    // check that the graph sorting works
+    cy.get('select').find('option').contains('subproject name').should('be.selected');
```

Comment:
Cypress recommends targeting elements by a `data-cy` attribute rather than the specific HTML element.  That way, the structure of the page can remain the same, while changing the HTML elements being used, without changing the test.

https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Reply:
Cool, great suggestion. Will do.